src/client/game.cpp

```diff
@@ -2594,12 +2594,13 @@ ClientDynamicInfo Game::getCurrentDynamicInfo() const
 {
 	v2u32 screen_size = RenderingEngine::getWindowSize();
 	f32 density = RenderingEngine::getDisplayDensity();
-	f32 gui_scaling = g_settings->getFloat("gui_scaling") * density;
-	f32 hud_scaling = g_settings->getFloat("hud_scaling") * density;
+	f32 gui_scaling = g_settings->getFloat("gui_scaling");
```

Comment:
Needs range limiting ( > 0) to not cause issues in `calculateMaxFSSize`. Furthermore: wouldn't it make sense to move these values to variables in `ClientDynamicInfo`? Currently there's a code duplication that has potential to go out of sync easily due to different code areas: game client and main menu.